sdk/storage/test-resources-post.ps1

```diff
@@ -44,4 +44,51 @@ Write-Verbose "CORS rule set for $storageAccountName"
 $datalakeStorageAccountName = $DeploymentOutputs['DFS_ACCOUNT_NAME']
 $context = New-AzStorageContext -StorageAccountName $datalakeStorageAccountName
 Set-AzStorageCORSRule -ServiceType 'Blob' -CorsRules $corsRules -Context $context
-Write-Verbose "CORS rule set for $datalakeStorageAccountName"
\ No newline at end of file
+Write-Verbose "CORS rule set for $datalakeStorageAccountName"
+
+# Run any post deployment script and set any additional keys to set in Env
+$AdditionalEnvKeys = @{}
+
+# Create SAS for storage account with additional permissions that are not supported by SRP deployment
+$storageAccount = $DeploymentOutputs['ACCOUNT_NAME']
+Write-Host "Creating SAS for storage account $storageAccount"
+$storageContext = New-AzStorageContext -StorageAccountName $storageAccount -StorageAccountKey $DeploymentOutputs['ACCOUNT_KEY']
+$storageSas = New-AzStorageAccountSASToken -ResourceType Service, Container, Object -Service Blob, File, Queue, Table -Permission "rwdxftlacup" -Context $storageContext
+$AdditionalEnvKeys["ACCOUNT_SAS"] = $storageSas
+
+$datalakeStorageAccount = $DeploymentOutputs['DFS_ACCOUNT_NAME']
+Write-Host "Creating SAS for datalake storage account $datalakeStorageAccount"
+$storageContext = New-AzStorageContext -StorageAccountName $datalakeStorageAccount -StorageAccountKey $DeploymentOutputs['DFS_ACCOUNT_KEY']
+$storageSas = New-AzStorageAccountSASToken -ResourceType Service, Container, Object -Service Blob, File, Queue, Table -Permission "rwdxftlacup" -Context $storageContext
+$AdditionalEnvKeys["DFS_ACCOUNT_SAS"] = $storageSas
+
+
+# Try to detect the shell based on the parent process name (e.g. launch via shebang).
```

Comment:
As we talked I'm Ok with this approach for now but I do think it is worth filing an issue in the tools repo about adding support for such scenarios because I'd hate to see this code duplicated a lot of times.

Reply:
File issue https://github.com/Azure/azure-sdk-tools/issues/1044 to track and add support in common place rather than in individual service script.